### ISSUE TITLE
Cant remove Deaths Odium

### ISSUE THREAD

**Issue by JahvontaiM** [TinyModularThings/UniqueEnchantments#73]
I lost some of my hearts and I used the death odium cookie and the phoenix's blessing. The cookie kills me instantly and keeps me at the low hearts I had before dying. Phoenix's blessing I used and set the damage value in the config for Deaths Odium to 1 to speed stuff up. I got the hearts back and when I did a death test it reset right back to the low hp I had before. How do you make this pain end without deleting the mod entirely because I am getting to that point.

**Comment by Xaikii** [comment]
First of all, I'd need to know which version you're on, of Unique Enchantments and Minecraft

**Comment by JahvontaiM** [comment]
Game version is 1.19.2 and the mod version is 4.0.2 if that is what 1.19.2-4.0.2 means. I have a work around to get my health back with the scaling health mod, but there is currently nowhere on the internet besides a reddit post and this website to really help people that have been stuck with this curse. I've not seen any videos on this mod to really know exactly what I am supposed to do, but I have tried eating the Enchanted cookie with the Deaths Odium 1 and 2, neither have worked. I went in to the in game config, lowered the damage value and did the Phoenix Blessing totem of undying and I mean it worked as intended but only for that life. After I died it reset my health back to missing 3 hearts.

**Comment by JahvontaiM** [comment]
I apologize for bringing up this curse and the problems that it is causing me and others, I know I am not the first person to talk about it and it must be very annoying, but I genuinely don't know why it refuses to go away. I am not running the mod on a server; I am using the Essentials mod and many others. Is it possibly the version I am in or am I missing something?

**Comment by Xaikii** [comment]
Ok, the game version matters because Death's Odium changed how it works between 1.12 and 1.16.5&1.19.2
As for the curse itself, once you've eaten the Cookie enchanted with Death's Odium, you're no longer affected by that curse at all. Everything you had from it, will be removed.
I've tested it now multiple times, I always regained my entire health after eating that cookie.
Also eating that cookie can not be reversed. Once you've eaten it you're no longer affected by Death's Odium for the rest of the playthrough. 

If this issue still persist though, then it's most-likely some kind of mod incompat, somebody else is messing with it.

**Comment by JahvontaiM** [comment]
I'll test it in another world some time and see if it's just something on my end. I'll record if there are any issues because like I said I ate the cookie and insta died, and it would flash my full health for a second then go back to 7 hearts.

**Comment by ikashana** [comment]
> 有时间我会在另一个世界测试它，看看它是否只是我这边的东西。如果有任何问题我会记录下来，因为就像我说的我吃了饼干然后 insta 死了，它会闪现我的健康一秒钟然后回到 7 颗心。

I also encountered this problem, can you tell me how did you solve it?

**Comment by JahvontaiM** [comment]
I didn't exactly solve the problem, but I went and downloaded a mod called scaling health to give myself the hearts back. The cookie temporarily gave me my full health back but when I died it would go right back down to  half a heart. Now I have some kinda PTSD when I see any armor set that has Deaths Odium on it😂

**Comment by Speiger** [comment]
@JahvontaiM thats the point.

**Comment by JahvontaiM** [comment]
Well that's very clear now but was not before, so this post is for anyone that runs into the same problem I did😉

**Comment by ikashana** [comment]
> I didn't exactly solve the problem, but I went and downloaded a mod called scaling health to give myself the hearts back. The cookie temporarily gave me my full health back but when I died it would go right back down to half a heart. Now I have some kinda PTSD when I see any armor set that has Deaths Odium on it😂

In the end, I could only temporarily solve the problem by changing the activated of the enchantment to false😂

**Comment by JahvontaiM** [comment]
Yeah doing that too helped out so none of my friends on my server ran into the same problem I did. I like the cool enchantments this mod adds but man it was almost game ending when Deaths Odium popped in😂

**Comment by a-Datana** [comment]
still the case

**Comment by musa18109** [comment]
hey there iam playing roughlike Adventures and Dungeons 2 and i cant enchant the cookie can anyone help me?
